code/modules/mob/living/carbon/human/species/yautja.dm

```diff
@@ -112,6 +112,10 @@
 	hive.hive_structure_types[XENO_STRUCTURE_NEST] = /datum/construction_template/xenomorph/nest
 	hive.hive_structures_limit[XENO_STRUCTURE_NEST]++
 
+	for(var/mob/living/carbon/Xenomorph/X in GLOB.living_xeno_list)
```

Comment:
More detailed var name! Also iirc there's a proper proc for this

Reply:
done, I took the proc from uh trap notifications and seems fine to me, most general hive announcement stuff has the stuff too from what I seen